persistence-dse-6.8/src/main/java/com/datastax/bdp/search/solr/Cql3SolrSecondaryIndex.java

```diff
@@ -18,7 +19,6 @@
 import org.apache.cassandra.schema.ColumnMetadata;
 import org.apache.cassandra.schema.IndexMetadata;
 import org.apache.cassandra.utils.concurrent.OpOrder;
-import org.jetbrains.annotations.Nullable;
```

Comment:
This started failing; apparently DSE-6.8.24 and before imported this alternate package. Everything else uses one from "javax.annotation.Nullable".